KebabCaseConvrter/Program.cs

```diff
@@ -1,12 +1,29 @@
 ﻿using System;
-
+using System.Text;
 namespace KebabCaseConvrter
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Enter your text: ");
+            string regularText = Console.ReadLine();
+
+            string[] words = regularText.Split(new char[] { ',', ' ', '.', ';', '-' }, StringSplitOptions.RemoveEmptyEntries);
+            var StringBuilder = new StringBuilder();
+
+            for (int i = 0; i < words.Length; i++)
```

Comment:
try to use 'foreach' instead. for loop is not always necessary , but it adds complexity for code readability. You can read about difference here https://docs.microsoft.com/en-gb/dotnet/csharp/language-reference/statements/iteration-statements